src/NHSD.BuyingCatalogue.Ordering.Services/OrderService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using NHSD.BuyingCatalogue.Ordering.Contracts;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+using NHSD.BuyingCatalogue.Ordering.Persistence.Data;
+
+namespace NHSD.BuyingCatalogue.Ordering.Services
+{
+    public class OrderService : IOrderService
+    {
+        private readonly ApplicationDbContext context;
+
+#pragma warning disable CS3001 // Argument type is not CLS-compliant
+        public OrderService(ApplicationDbContext context)
+#pragma warning restore CS3001 // Argument type is not CLS-compliant
+        {
+            this.context = context ?? throw new ArgumentNullException(nameof(context));
+        }
+
+        public async Task<DateTime?> GetCommencementDate(CallOffId callOffId)
+        {
+            return await context.Order
+                                .Where(o => o.Id == callOffId.Id)
+                                .Select(o => o.CommencementDate)
+                                .SingleOrDefaultAsync();
```

Comment:
Our code style is to only indent line continuations by one indent (aware it's not yet documented though), so this should be:

```c#
return await context.Order
    .Where(o => o.Id == callOffId.Id)
    .Select(o => o.CommencementDate)
    .SingleOrDefaultAsync();
```

Reply:
Updated indentation